### ISSUE TITLE
The results of diffpool direct training do not match the results in the paper

### ISSUE THREAD

**Issue by CigarOVO** [GalateaWang/TSGN-master#1]
When I use diffpool to train the ETHG2 dataset, the test set f1 input in TSGN format is only about 70%. Is there any step I missed?I got the test set f1 through the value of print in the code training step
`print(f'Epoch: {epoch:03d}, Train Loss: {train_loss:.4f}, '
          f'Val F1-score: {val_acc:.4f}, Test F1-score: {test_acc:.4f})`
Does the embeeding obtained after executing algorithms such as diffpool need to be further classified by the classifier?

**Comment by GalateaWang** [comment]
Thanks for your attention to this work. Our manuscript is under review. The source code will be updated later.
